### ISSUE TITLE
[BUG]: Can't unpause game

### ISSUE THREAD

**Issue by nafic01** [PCSX2/pcsx2#11790]
### Describe the Bug

I have set the "pause the game when controller disconnects" option and when it does reconnect and i press unpasuse from the game menu or the big picture menu idk, it doesn't work. I have to manually go out of fullscreen mode and unpause it from Settings\Unpause from the top menu

### Reproduction Steps

1) Somehow pause game
2) Try unpausing from the big picture or fullscreen menu

### Expected Behavior

The game is sopposed to unapause

### PCSX2 Revision

v2.0.2

### Operating System

Windows 11

**Comment by seta-san** [comment]
What game? Pal, ntsc?

**Comment by nafbul** [comment]
God of War, Im not talking about pausing the game, im talking about the actual emulator pause

**Comment by F0bes** [comment]
> can you join the discord , so you can get support :) ( see the readme for the link )

+1

**Comment by artdedeco** [comment]
Why was this closed? This still happens. Is this by design?

**Comment by JordanTheToaster** [comment]
> Why was this closed?

We cannot replicate this given the steps you can ask for help on our Discord or forums.

**Comment by artdedeco** [comment]
> > Why was this closed?
> 
> We cannot replicate this given the steps you can ask for help on our Discord or forums.

@JordanTheToaster I'm not the OP, but I can provide [better?] repro steps:

1. Enable the option to pause the game when controller disconnects
2. Start a game with either a DualSense or XSX controller (the only two I tested) connected. I tested the DS both via BT and cable; XSX  I tested only via wireless
3. Have the controller disconnect mid-game either manually (holding down the vendor button) or because the battery is dead. The VM should automatically pause.
4. Reconnect the controller or use any other input method to try and unpause the game. You'll see you are not able to do it unless you use the PCSX2 window menu bar

Video showing repro: https://1drv.ms/v/c/5c04e9c9d2116afb/EdRoQH9Z4o9Lh4dIoNLfeH8BrBQEXdeNzsfj9YURofOaSQ?e=rbZYoQ

Before seeing this issue and learning that there is at least the workaround of unpausing from the menu bar, I lost progress a few times in games before because of this bug. It's not at all obvious that you can do anything about it -- when it happens, it seems like it's bugged out -- and every time it happened to me I ended up restarting pcsx2.

Do note that if you have the 'vendor' button mapped to opening the pcsx2 pause menu, you'll not be able to repro this by manually disconnecting the controller through holding the vendor button: funnily enough, if you try that, it works as expected (the pause menu will open automatically and selecting "Resume game" actually unpauses it). If you want to repro by manually disconnecting, you need to have another button (eg. esc) mapped to opening the pause menu. Battery running out, on the other hand, still repros either way.

Can we please reopen the issue?

Test specs:

Game: repro'd on both SLES-53755 and SLUS-21216
PCSX2 2.4.0

OS:

Edition	Windows 11 Pro
Version	24H2
Installed on	‎27/‎10/‎2024
OS build	26100.4484
Experience	Windows Feature Experience Pack 1000.26100.128.0

Controller Settings:

<img width="1305" height="707" alt="Image" src="https://github.com/user-attachments/assets/e870e559-dc1e-461d-bf1d-4697ff1f8128" />

**Comment by artdedeco** [comment]
Folks, can we reopen this?

**Comment by F0bes** [comment]
Can someone else reproduce this?

**Comment by artdedeco** [comment]
I've left very clear instructions _and_ a video. If this won't be reopened, should I just go ahead and open a new issue instead?

**Comment by F0bes** [comment]
I'll re-open if someone else can reproduce. My question was more directed to the people who follow the repo activity
 Nothing needs to be done on your side.
If no one tries to reproduce it I'll open it then too.

**Comment by chaoticgd** [comment]
I can reproduce the game not unpausing when connecting the controller again and pressing resume in big picture, although that seems like more of a design issue (the paused/unpaused and pause menu shown/not shown states being separate) rather than a programming error.

**Comment by bslenul** [comment]
Yeah, basically the issue is more about being able to unpause the VM from the Big Picture menu than a bug.

**Comment by chaoticgd** [comment]
Actually why are those states separate? Could someone more familiar with how big picture mode works comment on this maybe?

**Comment by RedPanda4552** [comment]
Cannot repro. Booted BIOS with XSX controller, yanked USB, emulation paused, reconnected, hit spacebar, emulation resumed. Also remapped pause to the controller, still works fine.

**Comment by bslenul** [comment]
> hit spacebar

That's the whole issue, basically OP wants a way to be able to resume without having to hit any hotkey.

**Comment by artdedeco** [comment]
I'm not familiar with the codebase, so I don't know how those states are managed; FWIW, this is what I've gathered as for how they behave:

- Manually pause VM via toolbar: BP resume won't work;
- Pause via opening BP: pause is shown in the toolbar as well. Resume works and unpause "both";
- Pause via opening BP and then manually unpausing via toolbar: VM unpauses and game runs, but BP remains on top;
- Disconnect controller (this issue): same as pausing from toolbar
